modules/core/include/opencv2/core/private.cuda.hpp

```diff
@@ -99,6 +99,7 @@ namespace cv { namespace cuda {
     }
 
     CV_EXPORTS void syncOutput(const GpuMat& dst, OutputArray _dst, Stream& stream);
+    CV_EXPORTS void detachOutput(GpuMat& src, const GpuMat& dst, Stream& stream);
```

Comment:
I believe it should be a macro (to keep called function name and line in emitted warnings)

Reply:
https://github.com/opencv/opencv/commit/58dcd52e37783bc310e5b607cba08c0cb662da62 tried to resolve this review comment, however: https://github.com/opencv/opencv/pull/17863#issuecomment-661536808